File: src/OpenSBS.Engine/Server/ServerState.cs
namespace OpenSBS.Engine.Server
{
    public class ServerState
    {
        public bool IsReady { get; }
        public bool IsRunning { get; }

        public ServerState(bool isReady, bool isRunning)
        {
            IsReady = isReady;
            IsRunning = isRunning;
        }
    }
}

File: src/OpenSBS/client/src/store/server/serverState.ts
import DataEntryInfo from '../../models/dataEntryInfo';

export default interface ServerState {
    isReady: boolean;
    isRunning: boolean;
    lastTick: number;
    lastDeltaT: number;
    missions: DataEntryInfo[];
    spaceships: DataEntryInfo[];
}
